### ISSUE TITLE
Multiple references one feature Issue: From DRIP exploration

### ISSUE THREAD

**Issue by dwief-usgs** [internetofwater/ref_dams#6]
In some cases there is more than one ref_dam representing a unique feature (2 entries, 1 dam).  When this happens do we want to remove one of the reference dams or leave them both? Either way if we are fairly certain an entry is a duplicate we should cross reference IDs, right?.  Below are a few examples I ran into: 

- https://geoconnex.us/ref/dams/1078803 and https://reference.geoconnex.us/collections/dams/items/1087478.  In the NID these appear to have been reported by neighboring states and therefor have 2 entries but appear to be the same dam.  

- https://geoconnex.us/ref/dams/1090897, https://geoconnex.us/ref/dams/1051295, and https://geoconnex.us/ref/dams/1051297
These all appear to represent the same dam feature.

**Comment by ksonda** [comment]
Good idea!. @dblodgett-usgs this would be a use case for owl:sameAs right? or you want to be safer and schema:sameAs?

**Comment by dblodgett-usgs** [comment]
I'm not sure how much the sameAs version matters - owl:sameAs is the stronger of the two and the two really are the exact same physical feature -- so probably fine to go that route. schema:sameAs seems better for softer "webby" sameAs associations.
